docs/en/latest/plugins/response-rewrite.md

```diff
@@ -32,13 +32,25 @@ response rewrite plugin, rewrite the content returned by the upstream as well as
+| status_code | integer | optional    |         | [200, 598] | New `status code` to client, keep the original response code by default.                                                                                                                                                                                                                                                |
+| body        | string  | optional    |         |            | New `body` to client, and the content-length will be reset too.                                                                                                                                                                                                                                                         |
+| body_base64 | boolean | optional    | false   |            | Identify if `body` in configuration need base64 decoded before rewrite to client.                                                                                                                                                                                                                                       |
+| headers     | object  | optional    |         |            | Set the new `headers` for client, can set up multiple. If it exists already from upstream, will rewrite the header, otherwise will add the header. You can set the corresponding value to an empty string to remove a header. The value can contain Nginx variables in `$var` format, like `$remote_addr $balancer_ip`. |
+| vars        | array[] | optional    |         |            | A DSL to evaluate with the given ngx.var. See `vars` [lua-resty-expr](https://github.com/api7/lua-resty-expr#operator-list). if the `vars` is empty, then all rewrite operations will be executed unconditionally.                                                                                                      |
+| filters     | array[] | optional    |         |            | A group of filters that modify response body by replacing one specified string by another.                                                                                                                                                                                                                              |
+
+Only one of `body`, `filters` can be specified.
+
+### Attributes of filters
+
+| Name    | Type   | Requirement | Default | Valid          | Description                                                                                    |
+|---------|--------|-------------|---------|----------------|------------------------------------------------------------------------------------------------|
+| regex   | string | required    |         |                | match pattern on response body.                                                                |
+| scope   | string | required    | "one"   | "one","global" | substitution range.                                                                            |
```

Comment:
We should provide the doc of the nested field like `tls.client_cert` in https://github.com/apache/apisix/blob/master/docs/en/latest/admin-api.md.

And it is optional as people don't need to configure it.

Reply:
Had updated them.